scripts/r.in.srtm/r.in.srtm.py

```diff
@@ -270,7 +264,7 @@ def main():
     outf.close()
 
     try:
-        grass.run_command('r.in.gdal', input=bilfile, out=tileout)
+        grass.run_command('r.import', input=bilfile, out=tileout)
```

Comment:
```suggestion
        grass.run_command('r.import', input=bilfile, out=tileout, resample='bilinear')
```
... in addition: should we enforce the target resolution? But I don't know how to compute it as it depends on the map units etc.